schema/src/main/scala/org/constellation/schema/checkpoint/CheckpointBlockPayload.scala

```diff
@@ -0,0 +1,27 @@
+package org.constellation.schema.checkpoint
+
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
+import org.constellation.schema.Id
+import org.constellation.schema.signature.{Signable, Signed}
+
+import scala.language.implicitConversions
+
+case class FinishedCheckpointBlock(checkpointCacheData: CheckpointCache, facilitators: Set[Id]) extends Signable
+
+object FinishedCheckpointBlock {
+  implicit val encoder: Encoder[FinishedCheckpointBlock] = deriveEncoder
+  implicit val decoder: Decoder[FinishedCheckpointBlock] = deriveDecoder
+
+  implicit def toFinishedCheckpoint(a: FinishedCheckpointBlock): FinishedCheckpoint
+  = FinishedCheckpoint(a.checkpointCacheData, a.facilitators)
```

Comment:
Maybe FinishedCheckpointBlock could just wrap FinishedCheckpoint? Like FinishedCheckpointBlock(value: FinishedCheckpoint)?

Reply:
Why was repackaging needed actually?

Reply:
hmm, it would be `FinishedCheckpointBlock(FinishedCheckpoint)`. I think that rewriting this using `Signed` just for gossip is fine